Documentation/workflow/NativeDebuggingUsingAndroidStudio.md

```diff
@@ -0,0 +1,21 @@
+# Native Debugging of Mono using Android Studio
+
+With these steps it is possible to start a XA app with managed debugging in Visual Studio (on Mac and Windows) and native debugging of mono (with debugging symbols) using Android Studio.  
+
+Steps:
+1. Get the same github mono version that is in Android SDK
+2. Generate debug version of mono for android x86:
+    make -C sdks/builds build-android-x86 -j8 CONFIGURATION=debug
```

Comment:
Formatting: this should have a blank line above and below it, and be indented with 8 spaces.

Additionally, what is the current-working directory for this command?  Presumably it would be `external/mono`.

Finally, *is this actually necessary*?  If you go to the [xamarin-android-debug](https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-debug/104/) pipeline build, click on the "correct" build, then hit the **Azure Artifacts** link, the `bundle-v*.zip` link should contain a debug-build of mono.

(The problem here is determining what the "correct" link is, but there may be other ways to approach the problem.)

Reply:
> Formatting: this should have a blank line above and below it, and be indented with 8 spaces.
> 
> Additionally, what is the current-working directory for this command? Presumably it would be `external/mono`.
> 
> Finally, _is this actually necessary_? If you go to the [xamarin-android-debug](https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-debug/104/) pipeline build, click on the "correct" build, then hit the **Azure Artifacts** link, the `bundle-v*.zip` link should contain a debug-build of mono.
> 
> (The problem here is determining what the "correct" link is, but there may be other ways to approach the problem.)

And if you want to change something and continue debugging, trying to solve the bug.

Reply:
> And if you want to change something and continue debugging, trying to solve the bug.

Fair point, but there are cascading levels of complexity here.  Downloading a pre-existing `bundle*.zip` file is (fairly?) straightforward.  *Building mono* is an entirely different animal, and may be *impossible* on Windows.